src/ui/components/MobileHeaderPreview/MobileHeaderPreview.scss

```diff
@@ -0,0 +1,31 @@
+.mobile-preview-header {
+  display: none;
+  width: 100%;
+  background-color: transparent;
+  justify-content: space-between;
+  align-items: center;
+  padding: 13px 32px;
+  position: relative;
+  z-index: 22000;
```

Comment:
Hey @Sotatek-DukeVu this was just an information I left for you so that you are aware of what might happen.

Adding a comment about the z-index won't be necessary.